modules/acquia_cms_headless/modules/acquia_cms_headless_ui/acquia_cms_headless_ui.install

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/**
+ * @file
+ * Contains installation and update routines for acquia_cms_headless_ui.
+ */
+
+use Drupal\path_alias\Entity\PathAlias;
+
+/**
+ * Implements hook_install().
+ */
+function acquia_cms_headless_ui_install() {
+  \Drupal::messenger()->addStatus(__FUNCTION__);
+  // Update Aliases.
+  $aliases = [
+    '/admin/config/people/simple_oauth' => '/admin/access/settings',
+    '/admin/config/people/simple_oauth/oauth2_client' => '/admin/access/clients',
+    '/admin/people/roles' => '/admin/access/roles',
+    '/admin/config/people/simple_oauth/oauth2_token' => '/admin/access/tokens',
+    '/admin/people' => '/admin/access/users',
+    '/admin/structure/types' => '/admin/content-models/content',
+    '/admin/structure/media' => '/admin/content-models/media',
+    '/admin/structure/taxonomy' => '/admin/content-models/categories',
+    '/admin/config/people/accounts' => '/admin/content-models/users',
+    '/admin/structure/block/block-content' => '/admin/content/blocks',
+    '/admin/structure/block/block-content/types' => '/admin/content-models/blocks',
+  ];
+  foreach ($aliases as $path => $alias) {
+    PathAlias::create([
```

Comment:
I will work on updating this shortly.

Reply:
Good catch, this duplicates the path alias.  I would suggest we add a function to remove these on uninstall, since they are not needed nor valid if headless ui is uninstalled.

Reply:
Completed